DependencyInjection/Configuration.php

```diff
@@ -72,6 +72,7 @@ public function getConfigTreeBuilder()
                         ->scalarNode('stream_factory')->defaultNull()->end()
                     ->end()
                 ->end()
+                ->booleanNode('toolbar')->defaultTrue()->end()
```

Comment:
oh, you are right, yes. maybe add a description() saying that the toolbar is only active if the debug toolbar itself is active.